src/app/icd/ICDManagementServer.cpp

```diff
@@ -12,18 +12,18 @@ Status ICDManagementServer::RegisterClient(PersistentStorageDelegate & storage,
                                            uint64_t monitored_subject, chip::ByteSpan key,
                                            Optional<chip::ByteSpan> verification_key, bool is_admin)
 {
-    ICDMonitoringTable table(storage, fabric_index, GetClientsSupportedPerFabric());
+    ICDMonitoringTable table(storage, fabric_index, GetClientsSupportedPerFabric(), mSessionKeyStore);
```

Comment:
This and everythibg below is still naming this key store a "session key store"...

Reply:
replaced